Jenkinsfile

```diff
@@ -17,6 +17,17 @@ pipeline {
             }
         }
 
+        stage ('Static analysis') {
+            steps {
+                sh 'pylint katcomp --output-format=parseable --exit-zero > pylint.out'
```

Comment:
`katcomp`?

Reply:
The src directory. Probably could be more explicit written as `./katcomp`. Will test it out.